### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by igaryakqwe**

### CODE REVIEW

**Review by Markusplay — CHANGES_REQUESTED** [review]
image
No tg and tt icons on mobile

File: components/icons/tik-tok.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+const TikTok = () => {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      xmlnsXlink="http://www.w3.org/1999/xlink"
+      width="20"
+      height="20"
+      fill="none"
+      viewBox="0 0 20 20"
```
Comment (Markusplay) [inline components/icons/tik-tok.tsx:9]: icon's sizes should be flexiable

**Review by Markusplay — CHANGES_REQUESTED** [review]

**Review by Markusplay — CHANGES_REQUESTED** [review]
Icons on desktop and on moble should have different sizes

File: components/footer/components/copyrights.tsx
Comment (Markusplay) [inline components/footer/components/copyrights.tsx:1]: dumb component, return it to footer

**Review by Markusplay — CHANGES_REQUESTED** [review]

**Comment by igaryakqwe** [comment]
> Icons on desktop and on moble should have different sizes

done

**Comment by igaryakqwe** [comment]
> image No tg and tt icons on mobile

fixed

**Review by Markusplay — CHANGES_REQUESTED** [review]
Not adaptive for mobile

**Review by Markusplay — APPROVED** [review]

**Pull request merged**